test/test.js

```diff
@@ -9,25 +12,31 @@ const is = require("@slimio/is");
 const prettyJSON = require("../");
 
 async function executeNodeScript(path) {
-    const { stdout } = spawn(process.argv[0], [path]);
-    let str = "";
-    for await (const buf of stdout) {
-        str += buf;
-    }
+  const { stdout } = spawn(process.argv[0], [path]);
+  let str = "";
+  for await (const buf of stdout) {
+    str += buf;
+  }
 
-    str | 0;
-    return str;
-}
+  // eslint-disable-next-line no-unused-expressions
+  str | 0;
 
-test("prettyJSON must be a function", () => {
-    expect(is.func(prettyJSON)).toStrictEqual(true);
-});
+  return str;
+}
 
-test("prettyJSON of non-object must return undefined", () => {
-    expect(prettyJSON("hello")).toStrictEqual(undefined);
-});
+describe("getLocalLang/setLocalLang", () => {
+  it("prettyJSON must be a function", () => {
+    assert.strictEqual(is.func(prettyJSON), true);
+  });
 
-test("prettyJSON stdout must be the same", async() => {
-    const stdout = await executeNodeScript(join(__dirname, "fixtures", "01.js"));
-    expect(stdout).toMatchSnapshot();
+  it("prettyJSON : should generate an error", () => {
+    const expectedValue = "hello";
+    assert.throws(() => {
+      prettyJSON(expectedValue),
+      {
+        name: "Error",
+        message: `Payload -> ${expectedValue} should be object or array`
+      }
+    })
+  });
```

Comment:
```suggestion
  it("should throw when param is not object or array", () => {
    const expectedValue = "hello";
    assert.throws(
      () => prettyJSON(expectedValue),
      {
        name: "Error",
        message: `hello should be object or array.`
      }
    )
  });
```

Reply:
Your test is not OK.
`Payload -> ${expectedValue} should be object or array` -> you can try to edit the message to make your test not to pass, it will still pass because you pass a single function to `throws`